README.md

```diff
@@ -41,7 +41,7 @@ nvm use lts/hydrogen
 
 ## Install Solo
 
-* Run `npm clean install` or `npm ci`
+* Run `npm clean-install` or `npm ci`
 * Run `npm run build`
```

Comment:
as mentioned in the other PR, we should remove these two lines.  We want our Readme.md to be basic with a link to the doc, since it gets published to npmjs